lang/en.json

```diff
@@ -28,10 +28,12 @@
         "maintenance_message": "This store is currently unavailable due to maintenance. It should be available again shortly. We apologize for any inconvenience caused."
     },
     "brands": {
-        "no_products": "There are no products listed under this brand."
+        "no_products": "There are no products listed under this brand.",
+        "search_error": "There was a problem retrieving the products on this page. Please try again at a later time"
```

Comment:
We already have search error message for search, could it be used? It could be found at search.errorMessage field